src/commands/scanner/run.ts

```diff
@@ -116,6 +116,12 @@ export default class Run extends ScannerCommand {
 			description: messages.getMessage('flags.vceDescription'),
 			longDescription: messages.getMessage('flags.vceDescriptionLong'),
 			exclusive: ['json']
+		}),
+		engine:  flags.array({
```

Comment:
There should probably be a comment above this new flag indicating that it's relevant to the FilterTypes stuff defined in `ScannerCommand`. It also wouldn't hurt to add that comment above the relevant existing flags too.